contracts/token/ERC20/MigratableERC20.sol

```diff
@@ -0,0 +1,67 @@
+pragma solidity ^0.4.24;
+
+import "./ERC20.sol";
+import "./SafeERC20.sol";
+
+
+/**
+ * @title MigratableERC20
+ * @dev This strategy carries out an optional migration of the token balances. This migration is performed and paid for
+ * @dev by the token holders. The new token contract starts with no initial supply and no balances. The only way to
+ * @dev "mint" the new tokens is for users to "turn in" their old ones. This is done by first approving the amount they
+ * @dev want to migrate via `ERC20.approve(newTokenAddress, amountToMigrate)` and then calling a function of the new
+ * @dev token called `migrateTokens`. The old tokens are sent to a burn address, and the holder receives an equal amount
+ * @dev in the new contract.
```

Comment:
These inline docs explain how the contract works but not how it's used. I'd add a note saying at least that it's an abstract contract that must be combined with `StandardToken` or the user's own token implementation (with a `_mint` internal function), and if we have a link to a guide for how to use it I'd put that here as well!